packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/Controls.tsx

```diff
@@ -78,7 +78,9 @@ export const LocationDetailViewControls = (): React.JSX.Element => {
   const [state] = useControl('NAVIGATE');
   const { path } = state;
 
-  const [{ data, isLoading, hasError }, handleList] = useAction('LIST_LOCATION_ITEMS');
```

Comment:
Prettier change, though I'm not sure why it wasn't already there.

Reply:
I've been getting this a lot, there seems to be some non-deterministic behavior with our prettier setup

Reply:
Make sure your VSCode (not you personally, but all of us) is using the project's version of prettier and not a globally installed one